geoportailv3/static/js/maincontroller.js

```diff
@@ -211,6 +219,30 @@ app.MainController = function(
       $('app-themeswitcher #themes-content').collapse('hide');
     }
   }, this));
+
+  var selectInteraction = new ol.interaction.Select({
+    filter: goog.bind(function(feature, layer) {
+      return this.drawnFeatures.getArray().indexOf(feature) != -1;
+    }, this)
```

Comment:
For an unknown reason this doesn't work. I wonder if this is a bug in OL3. I'll have to check.